app/controllers/dashboard_controller.rb

```diff
@@ -57,7 +57,6 @@ def load_pq_with_counts(dashboard_state)
     @statuses                 = @questions.all.map(&:state).uniq
     @question_types           = @questions.where("question_type != 'Follow-up IWW'").map(&:question_type).uniq
```

Comment:
Does this "Follow-up IWW" need to be deleted?

Reply:
This value still exists in the database, so I have left this code in place.